build/charts/antrea/crds/bgppolicy.yaml

```diff
@@ -47,8 +47,8 @@ spec:
                 localASN:
                   type: integer
                   format: int32
-                  minimum: 64512
-                  maximum: 65535
+                  minimum: 1
+                  maximum: 4294967295
```

Comment:
I think we have a problem here because 4294967295 is MaxUint32 and the field is an int32

Also not sure how we can solve this. One possibility is to restrict the range to `[1, 65535]` for now, and change the field type from `int32` to `int64` in the next API version (e.g. `v1alpha2`). I do not believe we can update the type without also changing the API version.

Reply:
You just reminded me. We can only restrict the range to [1, 65535] in this PR. For extending the AS number to 32-bit, we can update it in the future with changing the API version.